pep-0678.rst

```diff
@@ -106,6 +106,19 @@ exception includes a note of the minimal failing example::
         +------------------------------------
 
 
+Non-goals
+---------
+``__note__`` is *not* intended to carry structured data.  If your note is for use by
+a program rather than display to a human, we recommend instead chosing a convention
+for an attribute like e.g. ``err._parse_errors = ...`` on the error or ``ExceptionGroup``
+(`discussion <https://discuss.python.org/t/accepting-pep-654-exception-groups-and-except/10813/26>`__,
+`discussion <https://bugs.python.org/issue46431>`__).
+
+As a rule of thumb, prefer `exception chaining <https://docs.python.org/3/tutorial/errors.html#exception-chaining>`__
+when the error is going to be re-raised or handled as an individual error; and prefer
+``__note__`` when you are collecting multiple exception objects to handle together or later.
```

Comment:
Here you could reference Raymond's post that kicked off the discussion about chaining as relevant history.